src/features/dotnetTest.ts

```diff
@@ -362,21 +327,17 @@ export default class TestManager extends AbstractProvider {
         // We support to styles of 'dotnet test' for debugging: The legacy 'project.json' testing, and the newer csproj support
         // using VS Test. These require a different level of communication.
 
-        const output = this._getOutputChannel();
-
-        output.show();
-        output.appendLine(`Debugging method '${testMethod}'...`);
-        output.appendLine('');
+        this._eventStream.post(new DotnetTestDebugStart(testMethod));
 
-        let { debugType, debugEventListener, targetFrameworkVersion } = await this._recordDebugAndGetDebugValues(fileName, testFrameworkName, output);
+        let { debugType, debugEventListener, targetFrameworkVersion } = await this._recordDebugAndGetDebugValues(fileName, testFrameworkName);
 
         return this._getLaunchConfiguration(debugType, fileName, testMethod, testFrameworkName, targetFrameworkVersion, debugEventListener)
             .then(config => {
                 const workspaceFolder = vscode.workspace.getWorkspaceFolder(vscode.Uri.file(fileName));
                 return vscode.debug.startDebugging(workspaceFolder, config);
             })
             .catch(reason => {
-                vscode.window.showErrorMessage(`Failed to start debugger: ${reason}`);
+                this._eventStream.post(new DebuggerStartFailure(reason));
```

Comment:
I could see this event name being confused with the actual debugger. Maybe something like `DebugDotNetTestStartFailure`?